lib/administrate/namespace.rb

```diff
@@ -21,7 +21,13 @@ def namespace_controller_paths
     end
 
     def all_controller_paths
-      Rails.application.routes.routes.map do |route|
+      Rails.application.routes.routes.select do |route|
+        if route.defaults[:administrate].nil?
+          true
+        else
+          route.defaults[:administrate]
+        end
+      end.map do |route|
```

Comment:
This certainly does the job but I'm trying to come up with a way to make this a little easier to unpack visually/mentally.

Pull the select out to its own method? Leave the map in here?